src/wallet.js

```diff
@@ -87,6 +87,7 @@ export function WalletProvider({ children }) {
       connectors={{
         fortmatic: { apiKey: getFortmaticApiKey() },
         portis: { dAppId: getPortisDappId() },
+        provided: { provider: window.cleanEthereum },
```

Comment:
I'd say this looks great. I actually quite like matching the `provided-connector` name here @bpierre , or do you have any strong reason to change the name to something like `ethereum-provider`?

Reply:
No that’s fine, let’s keep it this way!